spec/features/workflow/schedule_without_selection_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.feature "Attempting to schedule without confirming a review status" do
```

Comment:
Similar comment, I would remove "Attempting to" and rename the file to be clearer.

Reply:
Yep sure